analysis/webservice/algorithms/doms/ResultsStorage.py

```diff
@@ -177,21 +180,43 @@ def __insertResult(self, execution_id, primaryId, result, batch, insertStatement
 
         dataMap = self.__buildDataMap(data_dict)
         result_id = uuid.uuid4()
-        batch.add(insertStatement, (
-            result_id,
-            execution_id,
-            result["id"],
-            primaryId,
-            result["lon"],
-            result["lat"],
-            result["source"],
-            result["time"],
-            result["platform"] if "platform" in result else None,
-            result["device"] if "device" in result else None,
-            dataMap,
-            1 if primaryId is None else 0,
-            result["depth"]
-        ))
+
+        try:
+            batch.add(insertStatement, (
+                result_id,
+                execution_id,
+                result["id"],
+                primaryId,
+                result["lon"],
+                result["lat"],
+                result["source"],
+                result["time"],
+                result["platform"] if "platform" in result else None,
+                result["device"] if "device" in result else None,
+                dataMap,
+                1 if primaryId is None else 0,
+                result["depth"]
+            ))
+        except:
```

Comment:
This is not where the SQL query is actually executed -- the query is added to the batch and it's executed on line 172. So I don't think this would have the desired behavior

Reply:
The many error log statements were for me debugging this issue and can be removed/reduced.
Adding it to the batch prepares the SQL statement but doesn't execute it. It is in this preparation that the error occurred iirc.